app/src/main/java/com/example/vybe/Models/Vibe.java

```diff
@@ -32,7 +33,13 @@
      * @return A Vibe Enum
      */
     public static Vibe ofName(String name) {
-        return Vibe.valueOf(name.toUpperCase());
+        try {
+            return Vibe.valueOf(name.toUpperCase());
+
+        } catch (IllegalArgumentException e) {
+            return Vibe.BLANK;
```

Comment:
is it possible when we return a BLANK vibe that a toast also pops up about having an invalid vibe?

Reply:
Sure, that's a good idea

Reply:
Actually, to create a toast you need context and Vibe being a model class, doesn't have context